docs/conf.py

```diff
@@ -34,8 +33,13 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'sphinx_rtd_theme',
+    "sphinx.ext.intersphinx",
```

Comment:
for consistency, please use single quotes if the string in question does not contain single quotes.

(further instances of this heinous atrocity below..)

Reply:
i used black for consistency 😄